apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/gradingPointsSection.ejs

```diff
@@ -3,21 +3,25 @@
     <label for="js-<%= type %>-score-value-input-points-<%= context %>" class="js-manual-grading-points">
       <%= type_label %> Points:
     </label>
+    <% if (assessment_question.max_points) { %>
     <label for="js-<%= type %>-score-value-input-percentage-<%= context %>" class="js-manual-grading-percentage">
       <%= type_label %> Score:
     </label>
+    <% } %>
     <span class="float-right">
       <% if (!show_input) { %>
         <span class="js-manual-grading-points">
           <span class="js-<%= type %>-score-value-info">
-            <span class="js-value-<%= type %>-points"><%= Math.round(points * 100) / 100 %></span> / <%= max_points %>
+            <span class="js-value-<%= type %>-points"></span> / <%= max_points %>
           </span>
         </span>
+        <% if (assessment_question.max_points) { %>
         <span class="js-manual-grading-percentage">
           <span class="js-<%= type %>-score-value-info">
-            <span class="js-value-<%= type %>-percentage"><%= Math.round(points * 10000 / (max_points || assessment_question.max_points)) / 100 %></span>%
+            <span class="js-value-<%= type %>-percentage"></span>%
```

Comment:
The initialization of these values is unnecessary, since JS already updates them when the page is loaded. Keeping it blank for simplicity and to avoid inconsistencies between the formatting done here and in JS.

Reply:
Sounds good.

Unrelated to the current PR: I find this pattern of dynamically constructing class names (or attribute names, etc) to make the code harder to follow because we can't grep for names. I generally prefer to either have all the code written out multiple times, once for each possible class name, or to store the entire class name as a variable with the full class names in the code:
```
if (type == 'auto') {
    valuePercentageClass = 'js-value-auto-percentage';
} else {
    valuePercentageClass = 'js-value-manual-percentage';
}
```

Reply:
> Unrelated to the current PR: I find this pattern of dynamically constructing class names (or attribute names, etc) to make the code harder to follow because we can't grep for names. I generally prefer to either have all the code written out multiple times, once for each possible class name, or to store the entire class name as a variable with the full class names in the code:

Noted. Since this is not directly related to this PR I'll keep this as is and keep it in mind for future dev.